ipapython/ipautil.py

```diff
@@ -1104,14 +1104,17 @@ def reverse_record_exists(ip_address):
     return True
 
 
-def config_replace_variables(filepath, replacevars=dict(), appendvars=dict()):
+def config_replace_variables(filepath, replacevars=dict(), appendvars=dict(),
+                             removevars=None):
     """
     Take a key=value based configuration file, and write new version
-    with certain values replaced or appended
+    with certain values replaced, appended, ir removed
```

Comment:
Please fix the typo `ir removed`

Reply:
- Restore from a different version is not supported. ipa-server-upgrade should fix the cause, though.
- I have added ``systemd_daemon_reload``.